src/wallet/coinselection.cpp

```diff
@@ -50,8 +50,8 @@ struct {
  * The Branch and Bound algorithm is described in detail in Murch's Master Thesis:
  * https://murch.one/wp-content/uploads/2016/11/erhardt2016coinselection.pdf
  *
- * @param const std::vector<CInputCoin>& utxo_pool The set of UTXOs that we are choosing from.
- *        These UTXOs will be sorted in descending order by effective value and the CInputCoins'
+ * @param const std::vector<OutputGroup>& utxo_pool The set of UTXOs that we are choosing from.
+ *        These UTXOs will be sorted in descending order by effective value and the OutputGroups'
```

Comment:
In 18a6e09f0a50736a8113b6e6d918d93aa19d5a22

Maybe `The set of UTXO groups that we are choosing from` / `These UTXO groups will be sorted in[...]` is more accurate now.

Reply:
Done